platform/bootstrap/bootstrap.yml

```diff
@@ -300,36 +453,69 @@ Resources:
       Path: /
       Roles:
       - Ref: ClusterRole
-  BootstrapInstance:
-    Type: AWS::EC2::Instance
-    DependsOn: PublicSubnet1
+  AutoScalingGroup:
+    Type: AWS::AutoScaling::AutoScalingGroup
+    DependsOn:
+      - PublicSubnet1
+      - PublicSubnet2
+      - PublicSubnet3
+    UpdatePolicy:
+      AutoScalingRollingUpdate:
+        MaxBatchSize: 1
+        MinInstancesInService: !Ref ManagerSize
```

Comment:
the min instances in service shouldn't be the target size of the autoscaling group. It would be better to have that value minus one, but we don't want to calculate it here. I would leave it to 0, we have the batch size and the pause time already to give time to other instances to stabilize.